% !TEX root = ../../../../temp_manuscript.tex

\epigraphhead[450]{\epigraph{\say{We keep moving forward, opening new doors, and doing new things, because we're curious and curiosity keeps leading us down new paths.}}{--- \textup{Walt Disney}}}
\chapter[Differences in spatial distribution between WHO 2016 low-grade glioma molecular subgroups][Spatial distribution of WHO 2016 glioma subgroups]{Differences in spatial distribution between WHO 2016 low-grade glioma molecular subgroups}\label{chap:LGGLocation}

\begin{ChapterAbstract}
    \textbf{Background:} Several studies reported a correlation between anatomic location and genetic background of \cglspl{LGG}.
    As such, \gls{tumor} location may contribute to presurgical clinical decision-making.
    Our purpose was to visualize and compare the spatial distribution of different \cgls{WHO} 2016 gliomas, frequently aberrated single genes and DNA copy number alterations within subgroups, and groups of postoperative \gls{tumor} volume.

    \textbf{Methods:}
    Adult grade II glioma patients (\cgls{WHO} 2016 classified) diagnosed between 2003 and 2016 were included.
    \Gls{tumor} volume and location were assessed with semi-automatic software.
    All volumes of interest were mapped to a standard reference brain.
    Location heatmaps were created for each \cgls{WHO} 2016 glioma subgroup, frequently aberrated single \cglspl{CNV}, as well as heatmaps according to groups of postoperative \gls{tumor} volume.
    Differences between subgroups were determined using voxelwise permutation testing.

    \textbf{Results:}
    A total of 110 \cgls{IDH}-mutated \gls{astro} patients, 92 \cgls{IDH}-mutated and \acl{1p19qcodel} \gls{oli} patients, and 22 \cgls{IDH}-wildtype \gls{astro} patients were included.
    We identified small regions in which specific molecular subtypes occurred more frequently.
    \cgls{IDH}-mutated \cglspl{LGG} were more frequently located in the frontal lobes and \cgls{IDH}-wildtype \glspl{tumor} more frequently in the basal ganglia of the right hemisphere.
    We found no localizations of significant difference for single genes/\cglspl{CNV} in subgroups, except for loss of 9p in \glspl{oli} with a predilection for the left parietal lobes.
    More extensive resections in \cgls{LGG} were associated with frontal locations.

    \textbf{Conclusions:}
    \cgls{WHO} \gls{LGG} subgroups show differences in spatial distribution.
    Our data may contribute to presurgical clinical decision-making in \cgls{LGG} patients.

    \publishedas[True]{wijnenga2019differencesLGG}
\end{ChapterAbstract}

\section{Introduction}
Classification of diffuse gliomas is based on histological and molecular criteria according to the 2016 \cgls{WHO} classification of \glspl{tumor} of the central nervous system \autocite{louis2014international}.
Three major subtypes of diffuse \cgls{LGG} (grade II)  are recognized based on testing of 2 molecular markers: mutations of \cgls{IDH} 1 or 2 gene and combined deletion of chromosomal arms 1p and 19q.
Next to the \cgls{WHO} classification, (eloquent) location, size, presence of contrast enhancement, and delineation of the \gls{tumor} margins on \cgls{MRI} are important prognostic factors \autocite{pignatti2002prognostic, jakola2012low, capelle2013spontaneous, chang2008preoperative}.

Previous studies suggest that the anatomic location of a glioma is correlated with the molecular background of the \gls{tumor} \autocite{stockhammer2012idh1, goze2009lack, laigle2004correlations, metellus2010absence}.
If so, \gls{tumor} location may contribute to presurgical clinical decision-making and may provide a non-invasive marker for prediction of molecular subtype.
However, current evidence to support this is mainly derived from relatively small series which are often single molecular marker studies  \autocite{stockhammer2012idh1, goze2009lack, laigle2004correlations, metellus2010absence}.
For example, it was shown that loss of 1p is associated with a more frequent location in the frontal lobes and that \acl{1p19qcotion} is absent in insular \glspl{tumor} \autocite{goze2009lack, laigle2004correlations}.
In a series of 47 patients, \citeauthorref{metellus2010absence} showed that \cgls{IDH}-wildtype \cgls{LGG} are preferentially located in the insular region.
We recently reported on a cohort of resected \cgls{LGG} samples, wherein we found that \cgls{IDH}-wildtype \glspl{tumor} were more often located in eloquent areas \autocite{wijnenga2017impact}.
In that study, just as in many other previous studies on glioma location, we used the cerebral lobes as the location description factor, which is not accurate.
A recent study by \citeauthorref{tejada2018voxel} showed a location predilection for \cgls{IDH}-mutated \glspl{tumor} in the frontal lobes in the rostral extension of the lateral ventricles.
Despite these data, no large series have been described that visualize the anatomic location of \cgls{WHO} 2016 \cgls{LGG} subtypes in a voxel-based manner.
As the \cgls{WHO} 2016 criteria are more objective than the previous \cgls{WHO} 2007 criteria and result in a more refined and prognostic classification of \glspl{tumor}, it is possible that the spatial distributions of different \cgls{WHO} 2016 glioma subtypes are more distinct than previously shown for the classical histopathological classification \autocite{larjavaara2007incidence}.

Previously, we investigated the impact of extent of resection in \cgls{WHO} 2016 classified \cgls{LGG} and assessed \gls{tumor} volume location and volume in a semiautomatic manner \autocite{wijnenga2017impact}.
We used this cohort in the present study to visualize the spatial distribution of different \cgls{WHO} 2016 \cgls{LGG} subtypes, by creating voxel-based probability maps of \gls{tumor} location for every subtype.
Using this cohort also enabled us to visualize spatial distributions of \cglspl{LGG} according to the extent of resection.
These maps might be helpful in presurgical decision-making.

\section{Methods}
\subsection{Patient selection}
Adult patients (age \num{\geq 18} years) with histopathologically confirmed supratentorial grade II glioma were included.
Patients and samples were previously described in a study investigating the extent of resection in grade II glioma \autocite{wijnenga2017impact}.
All patients were treated in a single institute between 2003 and 2016: The Brain \Gls{tumor} Center at Erasmus MC Cancer Institute in Rotterdam, the Netherlands.
The study was approved by the medical ethics committee of Erasmus MC and conducted according to national and European regulations.

\subsection{Image acquisition and processing}
We used the preoperative \cgls{MRI} scans that were available from a routine clinical setting.
\cgls{MRI} sequence protocols varied per patient, as patients were diagnosed in several centers without the use of a uniform \gls{tumor} \cgls{MRI} protocol.
For inclusion in this study, at least a \cgls{FLAIR} or \cgls{T2} sequence needed to be available.
We did not use inclusion or exclusion criteria for voxel size and/or slice thickness.
We segmented preoperative and postoperative glioma lesions on \cgls{MRI} in a semi-automated fashion with the SmartBrush tool that is incorporated in Brainlab Elements (version 2.1.0.15).
With this tool, a three-dimensional (3D) \cgls{VOI} can be created by manually segmenting the lesion on 2 perpendicular slices, from which the software calculates a full 3D \cgls{VOI}, which was manually adjusted where necessary.
We used the \cgls{FLAIR} sequence when available (3D where possible); otherwise, the conventional \cgls{T2} sequence was used for segmentation.
All \gls{tumor}-related \cgls{FLAIR} and \cgls{T2} signal abnormalities were included in the segmentation.
The \cgls{FLAIR} and \cgls{T2} scans of all patients were affinely registered (preserving parallel lines and planes) to the \cgls{MNI} \cgls{ICBM} 152 nonlinear \cgls{T2} atlas version 2009a nonlinear symmetric \autocite{fonov2011unbiased, fonov2009unbiased} and the 3D \cglspl{VOI} were transformed accordingly for further analysis.
We registered all \cgls{MRI} scans using an automated algorithm based on maximization of mutual information \autocite{klein2009adaptive}, as implemented in the open-source SimpleElastix software (version 9dfa8cb) \autocite{marstal2016simpleelastix}.
All registrations were manually checked to assure proper alignment with the atlas, and adjusted where necessary.

\subsection{DNA extraction and \acrlong{NGS}}

Areas with high \gls{tumor} content were manually macrodissected from \cgls{FFPE} tissue slides as described previously \autocite{wijnenga2017impact}.
We used a targeted \cgls{NGS} panel to classify samples according to the \cgls{WHO} 2016 criteria, using an Ion Torrent Personal Genome Machine or Ion S5XL (Life Technologies).
The panel assesses mutational status of \cgls{IDH}1/2, TP53, FUBP1, PTEN, CIC, CDKN2A, NOTCH1, ATRX (whole gene) and hotspots of EGFR (exon 3 + 15), H3F3A (exon 2), PIK3CA (exon 10 + 21), BRAF (exon 11 + 15), and also \cglspl{CNV} of chromosome 1, 7, 9, 10, 12, and 19.
TERT promoter mutations (C228T and C250T) were assessed in a separate assay (SnaPshot).
Detailed methods were described previously \autocite{wijnenga2017impact, wijnenga2018prognostic}.

The following criteria for molecular classification were used:

\begin{itemize}

\item \Gls{oli}: \cgls{IDH}1 or \cgls{IDH}2 mutated and loss of heterozygosity consistent with co-deletion of the entire 1p and 19q chromosomal arms.

\item \cgls{IDH}-mutated \gls{astro}: \cgls{IDH}1 or \cgls{IDH}2 mutated.

\item \cgls{IDH}-wildtype \gls{astro}, with molecular features of \gls{glioblastoma} (according to recent cIMPACT-NOW update 3 \autocite{brat2018IMPACT}), in further text named as \say{\cgls{IDH}-wildtype \gls{astro}}: \cgls{IDH}1 or \cgls{IDH}2 wildtype and: TERT promoter mutation without \acl{1p19qcotion}, or loss of heterozygosity of chromosome 10q and imbalance of chromosome 7, or EGFR amplification.

\end{itemize}

\subsection{Statistical analysis}
We created \gls{tumor} location heatmaps for the different \cgls{WHO} subtypes by iterating over all voxels in the \cgls{MNI} atlas and counting the number of \gls{tumor} occurrences for each group in each voxel.
Via this procedure, we created location heatmaps for the \cgls{WHO} 2016 molecular subgroups.
Additional heatmaps were generated for glioma-specific single genes and chromosomes of interest (CIC, FUBP1, chromosome 7, and chromosomal arm 9p), as well as location heatmaps stratified for extensiveness of resection (4 groups of postoperative \gls{tumor} volumes: \SI{0}{\cubic\centi\metre}, \SIrange{0.1}{5.0}{\cubic\centi\metre}, \SIrange{5.1}{15.0}{\cubic\centi\metre}, \SI{>15.0}{\cubic\centi\metre}).

To test for differences in spatial distribution between \cgls{WHO} 2016 subgroups, we assessed the cluster-wise significance at the voxel-level between distributions, using permutation testing with threshold free cluster enhancement in the software package \cgls{FSL} Randomise \autocite{smith2009threshold, winkler2014permutation} (version 5.0.9, using \num{15000} permutations).
This approach corrects p-values for the familywise error in testing the many voxels, considering a corrected p-value of \num{<0.05} as statistically significant.
We first compared all \cgls{IDH}-wildtype \cglspl{LGG} with all \cgls{IDH}-mutated \cglspl{LGG}.
In a subsequent analysis, we assessed the statistical significance of location differences of \cgls{IDH}-mutated \gls{astro} and \gls{oli} separately.

\section{Results}

Our initial cohort consisted of \num{246} patients with confirmed \cgls{LGG}, for which \cgls{FFPE} material and \cgls{MRI} scans were available.
Twenty-two patients were excluded: \num{16} due to sequencing failure, \num{2} due to a preoperative contrast enhancement suggestive of \gls{glioblastoma}, and another 4 due to insufficient image quality.
Of the remaining \num{224} patients, \num{92} were \glspl{oli}, \num{110} were \cgls{IDH}-mutated \glspl{astro}, and 22 were \cgls{IDH}-wildtype \glspl{astro}.
Clinical characteristics of this cohort were consistent with \cgls{LGG} patient characteristics known from the literature and are shown in \cref{tab:LGG_location_characteristics}.

\subimport{Tables/}{clinical_characteristics.tex}

\subsection{Spatial distribution of WHO 2016 glioma subgroups}

Spatial distribution heatmaps for each \cgls{WHO} 2016 grade II glioma subgroup are shown in \cref{fig:LGG_location_heatmap_subgroups}.
Upon visual inspection, most \glspl{oli} were located in the frontal lobes and cortex, while \cgls{IDH}-mutated \glspl{astro} were more frequently located in the frontotemporal lobes and the insular region.
\cgls{IDH}-wildtype \glspl{astro} were more frequently located in the basal ganglia and rostral areas of the hemispheres.
In this cohort, \glspl{tumor} were slightly more frequently located in the left hemisphere, both for \cgls{IDH}-mutated \glspl{astro} and \glspl{oli}.

Statistical analysis of the spatial distributions indicated that \glspl{tumor} were more frequent in the anterior extensions of the lateral ventricles for \cgls{IDH}-mutated \cglspl{LGG} (\glspl{oli} and \cgls{IDH}-mutated \glspl{astro} combined) compared with \cgls{IDH}-wildtype \glspl{astro}.
The p-values per voxel are shown in \cref{fig:LGG_location_p_values_IDH_mut_vs_IDH_wt_mutated}.
With \cgls{IDH}-wildtype \glspl{astro} as reference category, our analysis indicated that \cgls{IDH}-wildtype \glspl{astro} are more frequently located in the basal ganglia of the right hemisphere (when compared with \cgls{IDH}-mutated \cglspl{LGG}) (\cref{fig:LGG_location_p_values_IDH_mut_vs_IDH_wt_wt}).

\begin{figure}
    \centering
    \begin{subfigure}[b]{\textwidth}
        \centering
        \resizebox{\textwidth}{!}{
            \subimport{Figures/}{heatmap_1p19q_codel.pgf}
        }
        \caption{\Gls{oli}, IDH-mutated 1p/19q co-deleted (\numbersamples{=92})}\label{fig:LGG_location_heatmap_IDH_mut_1p19q_codel}
    \end{subfigure}
    \begin{subfigure}[b]{\textwidth}
        \centering
        \resizebox{\textwidth}{!}{
            \subimport{Figures/}{heatmap_IDH_mut_1p19q_intact.pgf}
        }
        \caption{\Gls{astro}, IDH-mutated (\numbersamples{=110})}\label{fig:LGG_location_heatmap_IDH_mut_1p19q_intact}
    \end{subfigure}
    \begin{subfigure}[b]{\textwidth}
        \centering
        \resizebox{\textwidth}{!}{
            \subimport{Figures/}{heatmap_IDH_wt_1p19q_intact.pgf}
        }
        \caption{\Gls{astro}, IDH-wildtype (\numbersamples{=22})}\label{fig:LGG_location_heatmap_IDH_intact_1p19q_intact}
    \end{subfigure}
    \caption{Spatial distribution heatmaps of \acrshort{WHO} 2016 glioma subgroups. The color of a voxel corresponds with the number of \glspl{tumor} localized at that location, ranging from dark purple (low number) to bright orange (high number). The color bars on the right indicate the frequencies corresponding with the color per voxel;
    \refsubfigure{fig:LGG_location_heatmap_IDH_mut_1p19q_codel} location distribution of \gls{oli} shows most are located in the frontal lobes.
    \refsubfigure{fig:LGG_location_heatmap_IDH_mut_1p19q_intact} \acrshort{IDH}-mutated \gls{astro} show a distribution with most \glspl{tumor} located in or near the insular region.
    \refsubfigure{fig:LGG_location_heatmap_IDH_intact_1p19q_intact} \acrshort{IDH}-wildtype \glspl{astro} are more often located in midline region and basal ganglia}\label{fig:LGG_location_heatmap_subgroups}
\end{figure}

\begin{figure}
    \centering
    \begin{subfigure}[b]{\textwidth}
        \resizebox{\textwidth}{!}{
            \subimport{Figures/}{p_value_IDH_wt_vs_mutated_mutated.pgf}
        }
        \caption{Unique localizations for \acrshort{IDH}-mutated \acrshort{LGG}}\label{fig:LGG_location_p_values_IDH_mut_vs_IDH_wt_mutated}
    \end{subfigure}
    \begin{subfigure}[b]{\textwidth}
        \resizebox{\textwidth}{!}{
            \subimport{Figures/}{p_value_IDH_wt_vs_mutated_wt.pgf}
        }
        \caption{Unique localizations for \acrshort{IDH}-wildtype \acrshort{LGG}}\label{fig:LGG_location_p_values_IDH_mut_vs_IDH_wt_wt}
    \end{subfigure}

    \caption{Differences in location distribution between \cgls{IDH}-wildtype and mutated \acrlongpl{LGG}. Voxel-color indicates corrected p-value with color bar for scale}\label{fig:LGG_location_P_values}
\end{figure}

Direct comparison between \cgls{IDH}-mutated \glspl{astro} and \glspl{oli} showed a small area in the left frontal cortex where \glspl{oli} occurred significantly more frequently (\pvalue{< 0.05}), and a small region in the right temporal lobe where \cgls{IDH}-mutated \glspl{astro} (\cref{fig:LGG_location_oligo_astro_p_value}) occurred more frequently.
However, in a subsequent 3-group comparison, including \cgls{IDH}-wildtype \glspl{astro}, there were no locations of significant differences for \cgls{IDH}-mutated \glspl{astro} or \glspl{oli} individually, only for \cgls{IDH}-wildtype \glspl{astro} individually.

\begin{figure}[htbp]
    \centering
    \begin{subfigure}[b]{\textwidth}
        \resizebox{\textwidth}{!}{
            \subimport{Figures/}{p_value_1p19q_codeleted_vs_1p19q_intact_codeleted.pgf}
        }
        \caption{Unique localizations for \glspl{oli} compared with \acrshort{IDH}-mutated \glspl{astro}}\label{fig:LGG_location_p_values_1p19q_codeleted_vs_intact_codeleted}
    \end{subfigure}
    \begin{subfigure}[b]{\textwidth}
        \resizebox{\textwidth}{!}{
            \subimport{Figures/}{p_value_1p19q_codeleted_vs_1p19q_intact_intact.pgf}
        }
        \caption{Unique localizations for \acrshort{IDH}-mutated \glspl{astro} compared with \glspl{oli}}\label{fig:LGG_location_p_values_1p19q_codeleted_vs_intact_intact}
    \end{subfigure}
    \caption{Results of the \acrshort{FSL} Randomise test with \num{15000} permutations for statistical differences in location distribution between \glspl{oli} and \acrshort{IDH}-mutated \glspl{astro}.
    The color of a voxel is correlated with a p-value as indicated in the color bars on the right}\label{fig:LGG_location_oligo_astro_p_value}
\end{figure}

\subsection{Exploratory analysis of location predilection of a single gene and \acrshortpl{CNV}}

As an exploratory analysis, we also generated spatial distribution heatmaps of the additional genes and \cglspl{CNV} we tested with our dedicated \cgls{NGS} panel that were frequently mutated or aberrant.
For this, we analyzed CIC and FUBP1 mutations, and loss of chromosomal arm 9p for \gls{oli} (\cref{fig:LGG_location_heatmap_multi_genetics}).
We found no preferential locations for any of those molecular aberrations, except for loss of 9p (compared with \gls{oli} with intact 9p), which seemed to be more frequently located in the left parietal area (\cref{fig:LGG_location_p_value_9p}).

In \cgls{IDH}-mutated \gls{astro}, we created heatmaps of loss of chromosomal arm 9p and imbalance of chromosome 7, and found no preferential brain locations/voxels for either of those that showed p-values \num{<0.05} (\cref{fig:LGG_location_heatmap_chromosome_7_9}).

\begin{figure}[htbp]
    \centering
    \begin{subfigure}[b]{\textwidth}
        \resizebox{\textwidth}{!}{
            \subimport{Figures/}{heatmap_CIC_mut.pgf}
        }
        \caption{\acrshort{IDH}-mutated 1p/19q co-deleted \gls{oli}, CIC mutated (\numbersamples{=51})}\label{fig:LGG_location_heatmap_oli_cic_mutated}
    \end{subfigure}
    \begin{subfigure}[b]{\textwidth}
        \resizebox{\textwidth}{!}{
            \subimport{Figures/}{heatmap_CIC_wt.pgf}
        }
        \caption{\acrshort{IDH}-mutated 1p/19q co-deleted \gls{oli}, CIC wildtype (\numbersamples{=41})}\label{fig:LGG_location_heatmap_oli_cic_wt}
    \end{subfigure}
    \begin{subfigure}[b]{\textwidth}
        \resizebox{\textwidth}{!}{
            \subimport{Figures/}{heatmap_FUBP_mut.pgf}
        }
        \caption{\acrshort{IDH}-mutated 1p/19q co-deleted \gls{oli}, FUBP1 mutated (\numbersamples{=36})}\label{fig:LGG_location_heatmap_oli_fubp_mutated}
    \end{subfigure}
    \begin{subfigure}[b]{\textwidth}
        \resizebox{\textwidth}{!}{
            \subimport{Figures/}{heatmap_FUBP_wt.pgf}
        }
        \caption{\acrshort{IDH}-mutated 1p/19q co-deleted \gls{oli}, FUBP1 wildtype (\numbersamples{=56})}\label{fig:LGG_location_heatmap_oli_fubp_wt}
    \end{subfigure}
    \begin{subfigure}[b]{\textwidth}
        \resizebox{\textwidth}{!}{
            \subimport{Figures/}{heatmap_9p_loss_loss.pgf}
        }
        \caption{\acrshort{IDH}-mutated 1p/19q co-deleted \gls{oli}, loss 9p (\numbersamples{=23})}\label{fig:LGG_location_heatmap_oli_9p_loss}
    \end{subfigure}
    \begin{subfigure}[b]{\textwidth}
        \resizebox{\textwidth}{!}{
            \subimport{Figures/}{heatmap_9p_loss_intact.pgf}
        }
        \caption{\acrshort{IDH}-mutated 1p/19q co-deleted \gls{oli}, 9p intact (\numbersamples{=57})}\label{fig:LGG_location_heatmap_oli_9p_intact}
    \end{subfigure}
    \caption{Spatial distribution heatmaps of presence of individual gene or copy number alterations in the context of \gls{oli}}\label{fig:LGG_location_heatmap_multi_genetics}
\end{figure}

\begin{figure}[htbp]
    \centering
    \resizebox{\textwidth}{!}{\subimport{Figures/}{p_value_oligo_9p_loss.pgf}}
    \caption{Results of the \acrshort{FSL} Randomise test showing unique localizations for loss of 9p in the context of \gls{oli}}\label{fig:LGG_location_p_value_9p}
\end{figure}

\begin{figure}[htbp]
    \centering
    \begin{subfigure}[b]{\textwidth}
        \resizebox{\textwidth}{!}{
            \subimport{Figures/}{heatmap_IDH_mutated_astro_7_imbalance.pgf}
        }
        \caption{\acrshort{IDH}-mutated \gls{astro}, imbalance chromosome 7 (\numbersamples{=12})}\label{fig:LGG_location_heatmap_astro_chr_7_imbalance}
    \end{subfigure}
    \begin{subfigure}[b]{\textwidth}
        \resizebox{\textwidth}{!}{
            \subimport{Figures/}{heatmap_IDH_mutated_astro_7_intact.pgf}
        }
        \caption{\acrshort{IDH}-mutated \gls{astro}, chromosome 7 intact (\numbersamples{=93})}\label{fig:LGG_location_heatmap_astro_chr_7_intact}
    \end{subfigure}
    \begin{subfigure}[b]{\textwidth}
        \resizebox{\textwidth}{!}{
            \subimport{Figures/}{heatmap_IDH_mutated_chro9_loss.pgf}
        }
        \caption{\acrshort{IDH}-mutated \gls{astro}, loss 9p (\numbersamples{=12})}\label{fig:LGG_location_heatmap_astro_chr_9_loss}
    \end{subfigure}
    \begin{subfigure}[b]{\textwidth}
        \resizebox{\textwidth}{!}{
            \subimport{Figures/}{heatmap_IDH_mutated_chro9_intact.pgf}
        }
        \caption{\acrshort{IDH}-mutated \gls{astro}, 9p intact (\numbersamples{=70})}\label{fig:LGG_location_heatmap_astro_chr_9_intact}
    \end{subfigure}
    \caption{Spatial distribution heatmaps of presence of individual copy number alterations in the context of \acrshort{IDH}-mutated \gls{astro}}\label{fig:LGG_location_heatmap_chromosome_7_9}
\end{figure}

\subsection{Resection probability of \acrshort{LGG}}
For a previous study on the extent of resection in the same cohort, postoperative \gls{tumor} volumes were also assessed with the BrainLab Elements SmartBrush Tool.
We assigned patients into one of the 4 groups based on the postoperative \gls{tumor} volume:  \SI{0}{\cubic\centi\metre}, \SIrange{0.1}{5.0}{\cubic\centi\metre}, \SIrange{5.1}{15.0}{\cubic\centi\metre}, and \SI{>15.0}{\cubic\centi\metre} postoperative \gls{tumor} volume.
We generated location distribution heat maps stratified by these 4 groups, to investigate if there are preferential localizations for gross total resections.
Results are shown in \cref{fig:LGG_location_postop_volume}.
All \glspl{tumor} with a total resection (\SI{0.0}{\cubic\centi\metre} postoperative residue) were located in the frontal lobe.
Similarly, the majority of \glspl{tumor} with a low postoperative \gls{tumor} volume (\SIrange{0.1}{5.0}{\cubic\centi\metre}) were located in the frontal lobes.
\Glspl{tumor} with a postoperative volume of more than \SI{5.0}{\cubic\centi\metre} more frequently occurred in the insular region, temporal lobes, and in or near the primary sensory and motor cortex.

\begin{figure}[htbp]
\centering
\begin{subfigure}[b]{\textwidth}
    \resizebox{\textwidth}{!}{
        \subimport{Figures/}{heatmap_postop_0.pgf}
    }
    \caption{Total resection (\SI{0.0}{\cubic\centi\metre} postoperative \gls{tumor} volume) (\numbersamples{=33})}\label{fig:LGG_location_heatmap_postop_0}
\end{subfigure}
\begin{subfigure}[b]{\textwidth}
    \resizebox{\textwidth}{!}{
        \subimport{Figures/}{heatmap_postop_01_50.pgf}
    }
    \caption{\SIrange{0.0}{5.0}{\cubic\centi\metre} postoperative \gls{tumor} volume (\numbersamples{=54})}\label{fig:LGG_location_heatmap_postop_01_50}
\end{subfigure}
\begin{subfigure}[b]{\textwidth}
    \resizebox{\textwidth}{!}{
        \subimport{Figures/}{heatmap_postop_51_150.pgf}
    }
    \caption{\SIrange{5.1}{15.0}{\cubic\centi\metre} postoperative \gls{tumor} volume (\numbersamples{=40})}\label{fig:LGG_location_heatmap_postop_51_150}
\end{subfigure}
\begin{subfigure}[b]{\textwidth}
    \resizebox{\textwidth}{!}{
        \subimport{Figures/}{heatmap_postop_150.pgf}
    }
    \caption{\SI{>15.0}{\cubic\centi\metre} postoperative \gls{tumor} volume (\numbersamples{=97})}\label{fig:LGG_location_heatmap_postop_150}
\end{subfigure}
\caption{Spatial distribution heatmaps of \cgls{WHO} 2016 grade II glioma stratified according to postoperative \gls{tumor} volume.
 The color bars on the right indicate the frequencies corresponding with the color of the voxels}\label{fig:LGG_location_postop_volume}
\end{figure}

\section{Discussion}
In this study, we aimed to visualize and compare the spatial distribution of \cgls{WHO} 2016 grade II glioma subgroups.
By using advanced image processing analyses, we were able to generate accurate spatial distribution maps, especially compared with previous studies that were primarily based on location description/scores \autocite{stockhammer2012idh1, goze2009lack, laigle2004correlations, metellus2010absence}.
Our data indicate there are significant differences in spatial distribution patterns dependent on \cgls{IDH} status, with \cgls{IDH}-mutated \cglspl{LGG} more frequently located in the rostral extensions of the lateral ventricles, and \cgls{IDH}-wildtype \glspl{astro} more frequently in the basal ganglia of the right hemisphere.
Our data are in line with earlier observations and confirm there is a correlation between molecular background of a glioma and anatomic location \autocite{stockhammer2012idh1, goze2009lack, laigle2004correlations, metellus2010absence, tejada2018voxel}.
On the other hand, our data also indicate an overlap in anatomic location between \cgls{WHO} 2016 subgroups.

Upon visual inspection, a distinct pattern is clearly recognized between groups: most \glspl{oli} are located in the frontal lobes and cortex, while \cgls{IDH}-mutated \glspl{astro} are more frequently located in the frontotemporal and insular region.
However, the substantial overlap between \cgls{IDH}-mutated \glspl{astro} and \glspl{oli} can be appreciated as well.
This is also indicated by our voxel-cluster-based statistical analysis, wherein we find a significant predilection for \cgls{IDH}-mutated \cglspl{LGG} in the rostral extensions of the anterior lateral ventricles (\cgls{IDH}-mutated \glspl{astro} and \glspl{oli} grouped together), while we could not find regions significantly associated with either \cgls{IDH}-mutated \glspl{astro} or \glspl{oli} when we analyzed them as individual entities.
Although \glspl{oli} and \cgls{IDH}-mutated \glspl{astro} differ in clinical behavior (overall survival, sensitivity to chemotherapy) and are recognized as independent entities by the \cgls{WHO} classification, both entities share the \cgls{IDH} mutation.
It is suggested that the cell of origin for \cgls{IDH}-mutated gliomas is localized within the subventricular zone \autocite{sanai2005neural}.
If \glspl{oli} and \cgls{IDH}-mutated \glspl{astro} share the cell type of origin, this might explain the significant predilection of \cgls{IDH}-mutated \cglspl{LGG} in the rostral extensions of the anterior lateral ventricles, and the absence of a location difference between \cgls{IDH}-mutated \glspl{astro} and \glspl{oli}.

Compared with \cgls{IDH}-mutated \cglspl{LGG}, \cgls{IDH}-wildtype \glspl{astro} showed a distinct spatial distribution with more lesions located in the midline and basal ganglia.
This different spatial distribution is an interesting observation, as it shows that, in the setting of grade II gliomas, \cgls{IDH}-wildtype \glspl{astro} have a different anatomical and thus clinical presentation.
The spatial distribution explains the high percentage of biopsies in these patients we reported previously, as \glspl{tumor} in these locations are not eligible for safe resections \autocite{wijnenga2017impact}.
We also reported previously that these \glspl{tumor} often do not present with epilepsy, in contrast to \cgls{IDH}-mutated grade II gliomas \autocite{wijnenga2017impact}, and this might also be explained by their preferential, noncortical location.
On the other hand, it has also been postulated that high frequency of epilepsy in \cgls{IDH}-mutated gliomas is explained by mimicking the activity of glutamate on the NMDA receptor due to high levels of d-2-hydroxyglutarate \autocite{chen2017mutant}.

Upon visual inspection in this series, both \glspl{oli} and \cgls{IDH}-mutated \glspl{astro} were slightly more frequently located in the left hemisphere, especially in insular location.
An explanation might be a selection bias of patient referral because our center was one of the first in the Netherlands that performed awake craniotomies, which are performed for \glspl{tumor} located in presumed eloquent regions such as the left insular, frontal, and temporal region.

A current hot topic in glioma research is the development of non-invasive prediction of \cgls{WHO} 2016 classification of \gls{tumor} diagnosis based on preoperative \cgls{MRI} scans.
This would be very helpful for presurgical decision-making.
In a previous study, for example, we showed that minor postoperative \gls{tumor} residues have more impact on survival in \cgls{IDH}-mutated \glspl{astro} than in \glspl{oli} \autocite{wijnenga2017impact}.
Our data show that anatomic location could contribute to this non-invasive prediction, but requires the combination with other parameters to predict \cgls{WHO} subtype accurately.

We performed an exploratory analysis to assess the spatial distribution of other frequently reported mutations and \cglspl{CNV} in glioma.
Potential differences in spatial distribution might generate hypotheses or clues about the origin of glioma or specific subgroups and aggressiveness of certain glioma subtypes.
We found no specific spatial distributions for the tested aberrations however, except for loss of chromosome 9p in the context of \gls{oli}.
\Glspl{oli} with loss of 9p were significantly more frequently located in the left parietal area.
This finding needs to be confirmed in independent series before any assumptions regarding the relevance on the biological level can be made.

As a second aim we visualized the possible correlation between anatomic location of \cgls{LGG} and the extent of their resection.
State-of-the-art neurosurgical techniques including awake craniotomies were used in this cohort to achieve resections as extensive as possible in a safe way.
Upon visual inspection, we found that \cglspl{LGG} with no or very small postoperative \gls{tumor} residues (more extensive resections) were more frequently located in the frontal lobes, while \cglspl{LGG} with larger postoperative \gls{tumor} volumes were more frequently located in the insular and temporal regions.
This is an expected result, as the extent of resection is associated with anatomic location and also with proximity of eloquent areas of the brain \autocite{wijnenga2017impact}.
Our heatmaps provide an insightful visualization which might be helpful in surgical planning and in informing patients on resectability of an \cgls{LGG}.

Our study has several contributions compared with previous studies.
We used a relatively large and representative consecutive cohort of \cglspl{LGG}, which were all classified with \cgls{NGS} according to the integrated \cgls{WHO} 2016 criteria.
To the best of our knowledge, this is the first study that visualizes the spatial distribution of gliomas classified according to the \cgls{WHO} 2016 classification.
Also, we scored location in a voxel-based manner with the use of semi-automated segmentation software, which gives a far more accurate representation of location compared with a manual scoring of location.
Some limitations have to be addressed as well.
Our study was retrospective in nature, and the \cgls{MRI} protocols were not standardized.
Consequently, voxel size and slice thickness were not homogeneous in the cohort.
Images with large voxel size and slice thickness pose challenges for accurate segmentation and registration.
More subtle differences in location between groups might be missed.
However, our aim was not to find subtle differences, but clinically relevant differences between groups, and our cohort with heterogeneous \cgls{MRI} protocols is sufficient for this aim (and also reflects the \say{real-life} situation).
More importantly, the mapping of patient \cgls{MRI} scans with segmentation to the \cgls{MNI} standard brain can lead to distortion and a slight change of location on the standard brain compared with the original \cgls{MRI} scan.
This is especially relevant for relatively large lesions with mass effect, for example, \glspl{tumor} that compress the ventricles.
Because of this, our heatmaps erroneously showed some \glspl{tumor} to be located in the ventricles.
However, if we would have used a registration that perfectly mapped the \gls{tumor} to its position along the ventricle (instead of mapping it inside the ventricle), the \gls{tumor} would have been compressed.
As such the resulting mapping would misrepresent the actual \gls{tumor} volume.
Therefore, we have chosen to accept the erroneous mappings into the ventricles in favor of these volume effects.
To be as accurate as possible, we manually checked all registrations and corrected where necessary.
Furthermore, although this is a relatively large consecutive series of molecular-defined grade II glioma, a further expansion of our dataset is needed, as especially the group of \cgls{IDH}-wildtype \glspl{astro} is small.
We need larger numbers to confirm that the spatial distribution of \cgls{IDH}-wildtype \cgls{LGG} is indeed different compared with \cgls{IDH}-mutated \cgls{LGG}, as it is known that \glspl{glioblastoma} are also frequently located in the frontal lobes.
However, a recent study showed that, in the context of \gls{glioblastoma}, the frontal cortex is also significantly associated with the presence of \cgls{IDH} mutations \autocite{tejada2018voxel}.

In conclusion, \cgls{WHO} 2016 \gls{LGG} molecularly defined subgroups show both differences and similarities in spatial distribution, with \cgls{IDH}-mutated \cglspl{LGG} significantly more frequently located in the frontal lobes and \cgls{IDH}-wildtype \glspl{tumor} more frequently in the basal ganglia of the right hemisphere.
