% Following magic comments allow for compilation of root file
% !TEX root = ../../../../temp_manuscript.tex

\chapter{Introduction}
\section{Glioma}

When cells reproduce, genetic changes can occur that result in the cells starting to grow uncontrollably.
This uncontrollable growth of new cells is commonly known as cancer.
When the cancerous cells clump together, the single mass that they form is called a \glspl{tumor}.
\Glspl{tumor} can originate from and occur almost everywhere in the human body, and the location where the \gls{tumor} occurs is not always the same as the location from which the cancerous cells originate.
When the cancerous cells form a \gls{tumor} in the location from which they originate, the \gls{tumor} is called a primary \gls{tumor}.
Hence, primary brain \glspl{tumor} are \glspl{tumor} located in the brain that are formed by cells that mutated from (healthy) brain cells.

Since different types of brain cells exist, the cancerous cells can have mutated from these different cells.
Therefore, primary brain \glspl{tumor} are categorized by the type of brain cells from which they mutated, with \glspl{glioma} being the most prevalent \autocite{leece2017indicence}.
\Glspl{glioma} are \glspl{tumor} which cancerous cells mutated from \glspl{gcell}.
\Glspl{gcell} play a supporting role in the \acrlong{CNS} and are the most abundant cell type in the brain \autocite{jakel2017glial}.
Although \gls{glioma} have a low incidence compared to other cancers, around \per{1.7} globally \autocite{leece2017indicence}, they are one of the deadliest with a median survival of around ten months \autocite{hess2004gliomaincidence}.

\section{Categorization of glioma}
Historically \glspl{glioma} were categorized based solely on their histopathological appearance, that is, the appearance of the \gls{tumor} cells under a microscope.
Up to 2016, the \gls{WHO} recognized four different types of \gls{glioma}: \gls{astro}, \gls{oliastro}, \gls{oli}, and \gls{glioblastoma} (also known as \gls{GBM}).
This classification depended on the type of \glspl{gcell} that were visible in the \gls{tumor} tissue \autocite{louis2007who}.
Classifying \gls{glioma} based on their \gls{gcell} type is often called typing.
In addition to \gls{glioma} typing, \glspl{tumor} were assigned a grade to indicate their aggressiveness; this process is referred to as grading.
The grade was either II, III, or IV, with a grade IV \gls{glioma} being the most aggressive.
Although grade I \gls{glioma} exist, these are regarded as benign and as such are excluded from further consideration.
Grade II and grade III \gls{glioma} are often referred to as \gls{LGG}, and grade IV \gls{glioma} are referred to as \gls{HGG}.
There is some relation between the type and grade of a glioma; \gls{glioblastoma} are grade IV glioma, the most aggressive type, while \gls{astro}, \gls{oliastro}, and \gls{oli} are less aggressive and can be either grade II or grade III \footnote{Officially \gls{glioblastoma} are grade IV \gls{astro}, however in practice these are always referred to as \gls{glioblastoma} and \gls{astro} mean to the grade II/III ones.}.

However, this categorization based on the typing and the grading of the \gls{tumor} is suboptimal.
Firstly, the histopathological categorization and grading of the \glspl{tumor} are very observer-dependent \autocite{mittler1996gradingreliability, vandenbent2010interobserver}.
Since the clinical decision making depended on the categorization, this observer dependency could lead to an inadequate treatment of the \gls{tumor} \autocite{vandenbent2010interobserver}.
Secondly, within these categories, large differences between the survival of patients existed  \autocite{dubbink2015molecular}.
This suggest a different underlying mechanic at play that determines the aggressiveness of the \gls{tumor} than what the categorization based on grade and type accounts for.
For example, some grade III \gls{glioma} showed survival rates more characteristic of grade IV \gls{glioma} than other grade III \gls{glioma}.
It was found that this difference can be explained by the genetic features of the \gls{tumor} \autocite{dubbink2015molecular,eckel2015gliomagroups}.
Therefore, in 2016 the \gls{WHO} updated the categorization of \gls{glioma} to include these genetic features and depend less on the histopathology alone \cite{louis20162016}.
This update led to better patient stratification and more objective categorization of \gls{glioma} \autocite{molinaro2019geneticepidemiology}.

In the updated guidelines, the \gls{tumor} typing has become less relevant.
The histopathology now only needs to distinguish \gls{astro}, \gls{oliastro}, and \gls{oli} (\gls{LGG}) on the one hand and \gls{glioblastoma} (\gls{HGG}) on the other hand.
This greatly reduces the observer variability as most variability was observed in distinguishing between \gls{astro}, \gls{oliastro}, and \gls{oli} \autocite{mittler1996gradingreliability}.
Further categorization of the \gls{glioma} is now based on two genetic features:  the \gls{IDHmut} and \gls{codeletion}.
These are two genetic features that occur at specific parts of the genome but only occur in the genome of the \gls{tumor} and not in the genome of the healthy tissue.
When the \gls{IDH} gene is normal (the same as the \gls{IDH} gene of the healthy tissue), the \gls{tumor} is referred to as \gls{IDH} wildtype.
If the \gls{IDH} gene is mutated from the normal gene, the \gls{tumor} is a \gls{IDH} mutated \gls{tumor}.
The same holds for the \gls{codeletion}: if it is the same as the original gene it is \acl{1p19qint}, if it mutated from the original it is \acl{1p19qcodel}.

Within the \gls{LGG} (what was previously \gls{astro}, \gls{oliastro}, and \gls{oli}), the following categories now exist:
\begin{itemize}
    \item Diffuse \gls{astro}, \gls{IDH} wildtype
    \item Diffuse \gls{astro}, \gls{IDH} mutated
    \item \Gls{oli}, \gls{IDH} mutated and \acl{1p19qcodel}
\end{itemize}
No category exists for \gls{IDH} wildtype, \acl{1p19qcodel} \gls{glioma} since as it was found that all \acl{1p19qcodel} \glspl{tumor} are also \gls{IDH} mutated \autocite{labussi20101p19qcodeletedIDH}.
Although the terms \gls{astro} and \gls{oli} are still used in the updated guidelines, they no longer carry the same meaning as before, since their categorization is no longer based on the \gls{gcell} type but instead on the genetic features.
The \glspl{oli} have the best prognosis among the \gls{LGG}, with the diffuse \glspl{astro} \gls{IDH} wildtype showing the worst prognosis.
Due to the aggressiveness of \gls{IDH} wildtype \gls{astro}, it is even suggested that these are actually misclassified \gls{glioblastoma} \autocite{hartmann2010IDH1gbm, brat2018IMPACT}.
An overview of the \gls{WHO} 2016 guidelines and their relation to the previous guidelines is presented in \cref{fig:intro_glioma_categorization}.


\begin{figure}[hbt]
    \resizebox{0.8\textwidth}{!}{\subimport{Figures/}{WHO_2016_flowchart.pgf}}
    \centering
    \caption{The WHO 2016 categorizaton of glioma.}\label{fig:intro_glioma_categorization}
\end{figure}

The \gls{WHO} 2016 guidelines also categorize \gls{glioblastoma} based on the \gls{IDHmut} status, with \gls{IDH} mutated \gls{glioblastoma} having a better prognosis than \gls{IDH} wildtype \gls{glioblastoma}.
Although the guidelines currently only include the \gls{IDH} mutations status, there is another genetic feature that plays an important role in \gls{glioblastoma}: \gls{MGMT} methylation.
\gls{MGMT} methylation is an important genetic feature; patients with \gls{MGMT} methylated \glspl{tumor} survive longer than patients with \gls{MGMT} unmethylated \glspl{tumor} \autocite{martinez2007MGMT, gessler2018MGMT, weller2009molecularGBM}.
Thus, although not officially part of the most recent \gls{WHO}, guidelines the \gls{MGMT} methylation status is now often relevant in the clinical decision making \cite{molinaro2019geneticepidemiology}.

Not only do the genetic features determine the aggressiveness of the \gls{tumor}, but they also influence the \gls{tumor}'s response to certain treatments.
For example \gls{IDH} mutated \gls{glioma} respond better to radiotherapy than \gls{IDH} wildtype \gls{glioma} \autocite{juratli2015IDHtreatment}, and the \gls{codeletion} status and \gls{MGMT} methylation status might be predictors of the sensitivity to chemotherapy \autocite{idbaih2007markersresponse}.
Thus, it is important to know the genetic features of a \gls{glioma}, both for the prognosis of the patient and for the clinical decision making.

In current clinical practice, the genetic features are determined either from a biopsy, or a resection.
In the case of a biopsy, a small part of the \gls{tumor} is removed for analysis to decide on the further treatment of the \gls{tumor}.
Thus, biopsies are not part of the treatment but are merely used to decide on further treatment options.
Resection is part of the treatment process and involves surgically removing as much of the \gls{tumor} as possible.
Part of the removed \gls{tumor} can then be used for the genetic and histopathological analysis to decide on follow-up treatments.
Taking a biopsy or resecting the \gls{tumor} requires intrusive surgery, which not only puts an extra burden on medical experts but also comes with risks for the patient.
Therefore, it would be beneficial if the genetic features of a \gls{glioma} can be determined without the need for a biopsy or resection.
This is especially relevant in the case of a biopsy since the sole purpose of the biopsy is to obtain tissue that can be analyzed to help the clinical decision making.

A non-invasive method to determine the genetic features of the \gls{tumor} not only obviates the need for an intrusive surgical operation, but it can also provide relevant information earlier in the treatment process, which can aid in the clinical decision making.
In some cases, especially for \gls{IDH} mutated, \acl{1p19qcodel} \gls{glioma}, it might be better to follow a watch-and-wait approach since leaving \gls{tumor} untreated might have less of a negative impact than the potentially damaging effect of the treatment on the healthy tissue \autocite{vandenbent2012lggtreatment, welle2017EANO}.
Since \gls{MRI} is already part of the standard clinical pathway for glioma patients, it is the ideal candidate for a non-invasive way of identifying the genetic features of the \gls{tumor}.

\section{MRI image analysis of glioma}

\gls{MRI} is an imaging technique that uses the magnetic properties of protons, one of three elementary particles that form an atom, to create an image.
Although protons appear in all atoms, hydrogen atoms are of particular interest because they consist of a single proton, simplifying their magnetic properties and thus making it easier to use them for the imaging process.
Furthermore, a water molecule contains two hydrogen atoms and since the majority of the human body consists of water, hydrogen atoms (and thus protons) are abundant in the human body, especially in soft tissue.
An \gls{MRI} scanner manipulates the protons to form an image and can even manipulate them in different ways to obtain different imaging contrasts.

\gls{MRI} is routinely used in clinical care to distinguish between healthy and diseased tissue.
Shortly after the first introduction of \gls{MRI}, it was proposed that the method could be used to identify \glspl{tumor} \autocite{damadian1971tumor}.
After some improvements, the technique was first applied to the brain in 1980 \autocite{holland1980brain}, and in the same year MRI was first used to investigate a brain \gls{tumor} \autocite{hawkes1980NMRbrain}.
Since then, the quality of \gls{MRI} scans has vastly improved and different \gls{MRI} techniques have been developed that can image different tissue characteristics.
\cref{fig:intro_MR_example} shows some of the currently most popular imaging sequences, although many more exist.
In these scans, the \gls{tumor} can clearly be seen, which gives the clinician important information in a non-invasive way with minimal risk for the patient.
Therefore, \gls{MRI} is part of the standard-of-care for brain \gls{tumor} diagnosis and treatment decisions.

\begin{figure}[hbt]
    \centering
    \begin{subfigure}[b]{0.45\textwidth}
        \centering
        \includegraphics[width=\textwidth]{Figures/T1.png}
        \caption{}\label{fig:intro_T1}
    \end{subfigure}
    \begin{subfigure}[b]{0.45\textwidth}
        \centering
        \includegraphics[width=\textwidth]{Figures/T1GD.png}
        \caption{}\label{fig:intro_T1GD}
    \end{subfigure}

    \begin{subfigure}[b]{0.45\textwidth}
        \centering
        \includegraphics[width=\textwidth]{Figures/FLAIR.png}
        \caption{}\label{fig:intro_FLAIR}
    \end{subfigure}
    \begin{subfigure}[b]{0.45\textwidth}
        \centering
        \includegraphics[width=\textwidth]{Figures/T2.png}
        \caption{}\label{fig:intro_T2}
    \end{subfigure}

    \caption{Examples of \acrshort{MRI} scans with different contrasts. On some scans (\textbf{\protect\subref{fig:intro_FLAIR}}, \textbf{\protect\subref{fig:intro_T2}}) the \gls{tumor} appears as a bright lump, whereas on others (\textbf{\protect\subref{fig:intro_T1}}, \textbf{\protect\subref{fig:intro_T1GD}}) it appears darker than the surrounding healthy tissue. The different imaging contrasts provide clinically relevant information of the glioma; for example, the fact that scan \textbf{\protect\subref{fig:intro_T1}} shows a bright ring around a dark center suggests that this is a glioblastoma}\label{fig:intro_MR_example}
\end{figure}

\gls{MRI} scans are already routinely being used to get a first indication of the aggressiveness of the \gls{tumor}, mainly for \gls{tumor} grading \autocite{upadhyay2011MRIevaluation}.
With the increasing importance of genetic features of the \glspl{tumor}, research has focused on identifying imaging features that correlate with the genetic features of a \gls{tumor} \autocite{patel2017mismatch, smits2016imaging}.
These imaging features that correlate with an underlying physiological aspect, in this case the genetic features, are called imaging biomarkers.
The presence of these biomarkers proves the potential of \gls{MRI} as a non-invasive alternative to \gls{tumor} biopsy and resection.
However, their use in clinical practice is still limited.

A few limitations prevent the use of biomarkers for the genetic features of a \gls{glioma}.
Firstly, the biomarkers have to be relatively simple, because clinicians need to be able to quickly and easily (visually) extract them from the scans.
For example, most current clinically used biomarkers rely on 2D measurements (whereas the \gls{tumor} is of course 3D), or only consider a single point or small area of the \gls{tumor} instead of the whole \gls{tumor}.
As a result the information included in these biomarkers is limited, whereas using more information could lead to a better correlation with the physiological aspect of interest.
Secondly, \gls{MRI} is a qualitative and not a quantitative measurement, meaning that only the contrast of the scan and not its absolute value contains information \footnote{This is similar to food; it is easy to say that chocolate tastes better than Brussels sprouts, but one cannot say that chocolate tastes 5 \say{taste units} and Brussels sprouts tastes 1 \say{taste unit}.}.
Because of this, scans from the same patient from two different \gls{MRI} machines might have a similar visual appearance, but can result in different values for the same biomarker.
This complicates the use of these biomarkers as it is not possible to use their absolute value, which means that they can never be truly objective.
Thirdly, partly due to the previous point, the biomarkers are often vaguely defined.
Instead of defining a quantitative biomarker, qualitative biomarkers are used based on the interpretation of the visual appearance of the \gls{tumor}.
For example, a score can be assigned to how aggressive the \gls{tumor} looks.
This leads to biomarkers that are, similar to the typing and grading that was done before, observer-dependent.
Lastly, only a limited set of biomarkers is extracted and used for the correlation with the clinical outcome.
The method to correlates the biomarkers with the clinical outcome has to be simple to make it viable for the clinical experts to use.
Thus, although some promising biomarkers have been found, there is a need for new biomarkers and new methods that can include more information from the \gls{MRI} scan, are more objective, more consistent and, most importantly, do not put an additional burden on the clinicians.


\section{Goals of this thesis}

This thesis has three main goals;

\begin{enumerate}
\item Find more objective biomarkers for the genetic features of \gls{glioma} that experts can easily use
\item Provide tools and data to make research into new glioma biomarkers easier
\item Provide clinically relevant information of glioma that is normally obtained from invasive procedures from \gls{MRI} scans
\end{enumerate}

To achieve these goals, I focus on automated, computational methods.
Automated methods can carry out certain complex tasks more easily than humans, and do so in an objective, consistent way \footnote{Not all automated methods are necessarily consistent, some might contain a random element (purposefully or not). In this case, the potential inconstancy of the automated methods is negligible compared to the inconstancy of humans.}.

We define two types of automated methods: algorithmic methods and machine learning methods.
Algorithmic methods are methods where the process of determining an output from its inputs is explicitly defined by a human.
For example, the addition operator ($+$) is an algorithmic method, since we can now give two inputs (two numbers) and know exactly what our method will do with them.
In this way, we can create methods that can perform certain tasks very quickly which might be complex to do manually.
For example, these methods can very easily determine the 3D measurements of a \gls{tumor}, something that is hard to do by hand.
In this way, it is possible to extract more complicated biomarkers more easily and without any observer dependency.

The second type of automated methods are machine learning methods, which have become popular in recent years.
Machine learning methods are methods where the process of computing an output from its outputs is not explicitly defined but is left up to the method to figure out.
These methods do so by taking (a large amount of) data and try to find patterns in the input data can be correlated with the output data.
For example, a machine learning method could be given a long list of two inputs and one output, where the output is the addition of the two inputs.
It is then up to the machine learning method to figure out that it needs to sum to two inputs to get the correct output.
In this way, machine learning methods can learn relationships that were not yet known by clinical experts, making them a very versatile tool.

Using (a combination of) these methods allows us to achieve the goals outlined above and can solve the problems currently faced when using biomarkers in the clinic.
By taking the human aspect out of the extraction of biomarkers, the biomarkers can be more complicated and are not observer-dependent.
Furthermore, these automated methods can execute these tasks very fast, alleviating some of the burden placed on clinical experts.
When implemented properly, these methods are also widely applicable and do not require additional clinical expertise.
By using the strengths of automated methods to replace the current weaknesses of the manual methods, we can provide more and better information while allowing the strength of clinical experts to shine as well.

\section{Thesis outline}

This thesis is structured as follows.

\cref{chap:radiomics} provides an in-depth introduction of machine learning methods, introduces common terminology, and discusses some of the issues that can be encountered in machine learning research.

In \cref{chap:LGGLocation}, I use automated methods to find new biomarkers for \gls{LGG}, specifically looking for a relationship between the location of the \gls{glioma} in the brain and its genetic features.
The same methods are then used to establish a relationship between the genetic features of \gls{HGG} and their location in \cref{chap:HGGLocation}, now considering genetic features that are relevant for \gls{HGG}.
These chapters fit with goal 1.

In \cref{chap:LGG1p19q}, I further investigate potential biomarkers in \gls{LGG}, specifically for the \gls{codeletion} status.
These biomarkers are automatically extracted from the \gls{MRI} scans, to include more complex information than manually viable biomarkers.
I then use a machine learning method to link the biomarkers with the \gls{codeletion} status and compare the performance of this method with the performance of clinical experts.
This chapter fits with goals 1, 2 and 3.

Since machine learning methods require (large amounts of) structured data, in \cref{chap:DDS} I developed a method that automatically sorts large amounts of \gls{MRI} scans.
In this way it, is possible to quickly go from unstructured data from the clinical practice to structured data that can be used for research.
This chapter fits with goal 2.

Finally, in \cref{chap:discussion} I discuss the results from this thesis and explore possible future research directions.
I also provide an overview of the potential clinical applications that the methods developed in this thesis and other automated methods in the image analysis of glioma.

