% Following magic comments allow for compilation of root file
% !TEX root = ../../../../temp_manuscript.tex

\epigraphhead[450]{\epigraph{\say{Everything means something}}{--- \textup{Philip Pullman}, Lyra's Oxford}}
\chapter[Topographical mapping of 436 newly diagnosed IDH-wildtype glioblastoma with vs.\ without MGMT promoter methylation][Glioblastoma topographical mapping]{Topographical mapping of 436 newly diagnosed IDH-wildtype glioblastoma with vs.\ without MGMT promoter methylation}\label{chap:HGGLocation}


\begin{ChapterAbstract}
    \textbf{Introduction:} \cgls{MGMT} promoter methylation and \cgls{IDH} mutation status are important prognostic factors for patients with \cgls{glioblastoma}.
    There are conflicting reports about a differential topographical distribution of \cgls{glioblastoma} with vs.\ without \cgls{MGMT} promoter methylation, possibly caused by molecular heterogeneity in \cgls{glioblastoma} populations.
    We initiated this study to re-evaluate the topographical distribution of \cgls{glioblastoma} with vs.\ without \cgls{MGMT} promoter methylation in light of the updated \cgls{WHO} 2016 classification.

    \textbf{Methods:} Preoperative \acrlong{T1C} and \acrlong{T2}/\acrlong{FLAIR} \acrlong{MRI} scans of patients aged 18 year or older with \cgls{IDH}-wildtype \cgls{glioblastoma} were collected.
    \Glspl{tumor} were semi-automatically segmented, and the topographical distribution between \cgls{glioblastoma} with vs.\ without \cgls{MGMT} promoter methylation was visualized using frequency heatmaps.
    Then, voxel-wise differences were analyzed using permutation testing with threshold free cluster enhancement.

    \textbf{Results:} Four hundred thirty-six \cgls{IDH}-wildtype \cgls{glioblastoma} patients were included; 211 with and 225 without \cgls{MGMT} promoter methylation.
    Visual examination suggested that when compared with \cgls{MGMT} unmethylated \cgls{glioblastoma}, \cgls{MGMT} methylated \cgls{glioblastoma} were more frequently located near bifrontal and left occipital periventricular area and less frequently near the right occipital periventricular area.
    Statistical analyses, however, showed no significant difference in topographical distribution between \cgls{MGMT} methylated vs. \cgls{MGMT} unmethylated \cgls{glioblastoma}.

    \textbf{Conclusions:} This study re-evaluated the topographical distribution of \cgls{MGMT} promoter methylation in 436 newly diagnosed \cgls{IDH}-wildtype \cgls{glioblastoma}, which is the largest homogenous \cgls{IDH}-wildtype \cgls{glioblastoma} population to date.
    There was no statistically significant difference in anatomical localization between \cgls{MGMT} methylated vs.\ unmethylated \cgls{IDH}-wildtype \cgls{glioblastoma}.
    \publishedas[True]{incekara2020topographicalmaping}
\end{ChapterAbstract}

\section{Introduction}

Patients with \cgls{glioblastoma} have a poor prognosis with a median overall survival of 15 months, despite standard of care consisting of safe and maximal surgical resection, followed by chemo and/or radiotherapy \autocite{stupp2005radiotherapy}.
This prognosis varies based on factors such as age, \cgls{KPS}, extent of resection, and molecular markers, in particular \cgls{IDH} mutation and \cgls{MGMT} promoter methylation status \autocite{gessler2018MGMT}.

\cgls{MGMT} is a DNA repair enzyme, which is expressed by the \cgls{MGMT} gene located on chromosome 10q26.
Promoter methylation of this gene reduces \cgls{MGMT} protein expression and consequently decreases DNA repair and increases alkylating chemotherapy induced \gls{tumor} death.
Therefore, patients with \cgls{MGMT} methylated \cgls{glioblastoma} are more sensitive to neo-adjuvant temozolomide than those without \cgls{MGMT} methylated \cgls{glioblastoma}.
\cgls{MGMT} is methylated in around \per{50} of patients with newly diagnosed \cgls{glioblastoma} \autocite{hegi2005MGMT}.

There are conflicting results in the published literature on a possible differential topographical distribution of \cgls{glioblastoma} with vs.\ without \cgls{MGMT} promoter methylation \autocite{smits2017imaging}.
\citeauthorref{ellingson2012anatomic} suggested that when compared with those without \cgls{MGMT} promoter methylation, \cgls{glioblastoma} with \cgls{MGMT} methylation are more frequently located in the left temporal lobe and less frequently in the right temporal lobe.
However, other studies found the reverse lateralization pattern \autocite{wang2014anatomical} or did not find any lateralization at all \autocite{carillo2012relationship, eoli2007methylation, han2018structural}.
These conflicting results could be ascribed to heterogeneity of molecular subtypes of \cgls{glioblastoma} in the studied populations, for instance when \cgls{IDH}-wildtype \cgls{glioblastoma} are mixed with the genetically, and prognostically distinct \cgls{IDH}-mutated \cgls{glioblastoma}, or to variation in statistical methods that were used across studies.
Therefore, the question whether \cgls{glioblastoma} with vs.\ without \cgls{MGMT} promoter methylation have a different anatomical localization remains unanswered.
In light of the updated \cgls{WHO} 2016 classification \autocite{louis20162016}, a molecularly homogenous \cgls{glioblastoma} population must be used to re-evaluate the topographical distribution of \cgls{MGMT} methylated vs.\ unmethylated \cgls{glioblastoma}.

Therefore, we have initiated this study to re-evaluate the topographical distribution of \cgls{glioblastoma} with vs.\ without \cgls{MGMT} promoter methylation in the largest homogenous \cgls{IDH}-wildtype \cgls{glioblastoma} population to date.

\section{Methods}
\subsection{Patient inclusion}

All consecutive patients aged 18 years or older newly diagnosed with a contrast-enhancing and histopathologically confirmed \cgls{glioblastoma} \gls{IDH}-wildtype who underwent \gls{tumor} resection or biopsy between January 2011 and May 2018 at the Erasmus MC, University Medical Center Rotterdam, or Haaglanden MC were retrospectively included in this study.
Patients were eligible if preoperative  \gls{T1C} and \gls{T2}/\gls{FLAIR} \gls{MRI} scans as well as molecular data on \gls{IDH} mutation and \gls{MGMT} methylation status were available.
Recurrent \cgls{glioblastoma} or confirmed \gls{IDH}-mutated \cgls{glioblastoma} were excluded.
The study design was approved by the Medical Ethical Committee of Erasmus MC and Haaglanden MC\@.
The study was performed in accordance with the 1964 Helsinki Declaration and its later amendments or comparable ethical standards.

\subsection{Image acquisition, \gls{tumor} segmentation, and registration}
From clinical preoperative \gls{MRI} scans, which were obtained according to clinical brain \gls{tumor} protocols on either a \SI{1.5}{\tesla} or \SI{3.0}{\tesla} scanner, \gls{T1C} and \gls{T2}/\gls{FLAIR} images were collected.
For \cgls{glioblastoma} segmentation, we first imported both the \gls{T1C} and \gls{T2}/\gls{FLAIR} scans into BrainLab (BrainLab, Feldkirchen, Germany, version 2.1.0.15).
We semi-automatically segmented all \gls{tumor}-related contrast-enhancement (including the central necrotic part, if present) using the SmartBrush tool in Brainlab Elements and manually adapted the segmentation if needed.
We then used the \gls{T2}/\gls{FLAIR} scan to semi-automatically segment all \gls{tumor}-related non-enhancing hyperintense abnormalities (extra-lesional hemorrhage was excluded).

All \gls{tumor} segmentations were then registered to the \gls{MNI} International Consortium for Brain Mapping 152 non-linear, symmetric atlas 2009a \autocite{fonov2011unbiased, fonov2009unbiased}.
The \gls{T1C} scans were registered to the \acrlong{T1} atlas and the \gls{T2}/\gls{FLAIR} scans to the \gls{T2} atlas.
Registration was done using SimpleElastix (version 72b7e81), based on a mutual information metric using an affine registration \autocite{marstal2016simpleelastix}.
The resulting transformation parameters were used to transform the 3D segmentations to the atlas space.
Registration results were visually checked to ensure that for all cases, the registered masks lay entirely within the brain mask of the atlas.
No adjustments were made to the initial registration settings for individual patients.
We created voxel-wise frequency maps for all \cgls{glioblastoma} combined, and frequency difference maps of \cgls{glioblastoma} with vs.\ without \gls{MGMT} promoter methylation.

\subsection{Molecular analysis}

\Gls{tumor} tissue samples were obtained from patients through surgical resection or biopsy.
Histopathological examination was performed by neuropathologists.
DNA was extracted from microdissected \gls{FFPE} tissue fragments by proteinase K digestion for 16 h at \SI{56}{\celsius} in the presence of \per{5} Chelex 100 resin and used after inactivation of proteinase K and removal of cell debris and the Chelex resin.
\gls{IDH} mutational analysis was assessed with Sanger sequencing and targeted \gls{NGS} analysis.
Sanger sequencing of PCR-amplified fragments from \gls{IDH}1 and \gls{IDH}2 mutational hot spots was essentially performed as previously described \autocite{vandenbent2013interlaboratory}.
M13-tailed primers for PCR amplification of IDH1 codon 132 were forward \texttt{5'-TCTTCAGAGAAGCCATTAT-3'} and reverse \texttt{5'-GCAAAATCACATTATTGCCAAC-3'}, for IDH2 codon 140, forward {5'-GGCTGCAGTGGGACCACTAT-3'} and reverse \texttt{5'-TTGGTCCAGCCAGGGACTAG-3'}, and for IDH2 codon 172, forward \linebreak[4]\texttt{5'-ACATCCTGGGGGGGACTGTC-3'} and reverse \texttt{5'-GACAAGAGGATGGCTAGGCG-3'}.
The M13-tail for the forward primers was: \texttt{5'-TGTAAAACGACGGCCAGT-3'} and for the reverse primers: \texttt{5'-CAGGAAACAGCTATGACC-3'}.
After initial denaturation at \SI{95}{\celsius} for 3 min, 35 cycles of \SI{95}{\celsius} for 15 s, \SI{60}{\celsius} for 15 s, and \SI{72}{\celsius} for 15 s were performed, followed by 7 min at \SI{72}{\celsius}.
Subsequent sequence analyses of the PCR products was carried out with M13 forward and reverse primers on a 3730 XL Genetic Analyzer (Applied Biosystems, Foster City, CA, USA).

Targeted \gls{NGS} was performed by semiconductor sequencing with the Ion Torrent platform using supplier's materials and protocols (Thermo Fisher Scientific) with a dedicated panel for detection of glioma-specific aberrations, including \gls{IDH}1 and \gls{IDH}2 hot spot mutations essentially as previously described \autocite{dubbink2015molecular}.
Library and template preparations were performed consecutively with the AmpliSeq Library Kit 2.0--384 LV and the Ion PGM Template OT2 200 kit.
Sequencing was performed with the Ion PGM Sequencing 200 Kit v2 on an 318v2 chip with the PGM system.
Data were analyzed with the Torrent variant caller (Thermo Fisher Scientific), and variants were annotated in a local Galaxy pipeline using ANNOVAR\@.
Data were collected during several years using different glioma panels.
Sequenced areas of \gls{IDH}1 codon 132 and \gls{IDH}2 codons 140 and 172 are given in the supplementary data of \citeauthorref{dubbink2015molecular}.

\gls{MGMT} promoter methylation status was assessed by methylation-specific PCR essentially as described by \citeauthorref{esteller2000inactivation}.
Bisulfite conversion and subsequent purification are performed with the EZ DNA Methylation-Lightning Kit (Zymo Research) according to the supplier's protocol.
Methylation-specific PCR was performed with primers specific for either methylated or the modified unmethylated DNA\@.
Converted primer sequences for unmethylated DNA were forward \linebreak[4]\texttt{5'-TTTGTGTTTTGATGTTTGTAGGTTTTTGT-3'} and reverse \linebreak[4]\texttt{5'-AACTCCACACTCTTCCAAAAACAAAACA-3'}, and for the methylated reaction, forward \linebreak[3]\texttt{5'-TTTCGACGTTCGTAGGTTTTCGC-3'} and reverse \linebreak[3]\texttt{5'-GCACTCTTCCGAAAACGAAACG-3'}\@.\linebreak[4]
PCR was performed after initial denaturation at \SI{96}{\celsius} for \SI{5}{\minute}, 40 cycles of \SI{92}{\celsius} for \SI{45}{\second}, \SI{59}{\celsius} for \SI{65}{\second}, and \SI{72}{\celsius} for \SI{45}{\second}, followed by \SI{7}{\minute} min at \SI{72}{\celsius}.
Five microliter of each \SI{15}{\micro\liter} methylation-specific PCR product was loaded onto a \per{1.5} agarose gel stained with GelRed (Biotium) and examined under ultraviolet illumination.
SW48 cell line DNA and tonsil DNA was used as a positive control for methylated and unmethylated alleles of \gls{MGMT}, respectively.
Controls without DNA were used for each set of methylation-specific PCR assays.

\subsection{Statistical analysis}
We first tested the differences between preoperative enhancing and non-enhancing \gls{tumor} volumes as well as their ratio with the Kruskal-Wallis test.
We mapped the anatomical localization of all \gls{MGMT} methylated and unmethylated \cgls{glioblastoma} by iterating over all voxels in the \gls{MNI} atlas and counting the number of \gls{tumor} occurrences for each group in each voxel.
To test for differences in spatial distribution between \cgls{glioblastoma} with vs.\ without \gls{MGMT} promoter methylation, we assessed the cluster-wise significance at the voxel-level between distributions, using permutation testing with threshold free cluster enhancement in the software package \gls{FSL} randomise (version 5.0.9, using \num{10000} permutations) \autocite{smith2009threshold, winkler2014permutation}.
We also performed the same analysis with correction for age as a potential confounder by determining age for each patient at the time of the \gls{MRI} scan.
We then calculated the difference between each patient's age and the average age of all patients included in the study, which was added to the experimental setup for \gls{FSL} randomise.
\gls{FSL} randomise corrects p-values for the family-wise error in testing multiple voxels, considering a corrected p-value \num{< 0.05} as statistically significant.


\section{Results}


In total, 769 patients with newly diagnosed, contrast-enhancing \cgls{glioblastoma} were screened, of whom we excluded 333 patients: 22 were excluded due to \gls{IDH} mutation and 311 were excluded due to insufficient or missing molecular data on \gls{IDH} mutation or \gls{MGMT} methylation status.
Final analysis included 436 patients with \gls{IDH}-wildtype \cgls{glioblastoma} (see flowchart, \cref{fig:HGG_location_flowchart}); 211 with and 225 without \gls{MGMT} promoter methylation.
Three hundred forty patients had undergone a surgical \gls{tumor} resection and 96 a diagnostic biopsy.
In all patients, preoperative \gls{T1C} \gls{MRI} scans were available; in 90 patients, \gls{FLAIR} scans, and in 346 patients \gls{T2} scans were available.
When compared with \gls{MGMT} unmethylated \cgls{glioblastoma}, \gls{MGMT} methylated \cgls{glioblastoma} had a significantly higher ratio of non-enhancing vs.\ contrast-enhancing volume: \num{2.09} (inter quartile range \num{2.6}) and \num{2.5} (inter quartile range \num{3.3}), p \num{=0.045}, respectively.
Patient and \gls{tumor} characteristics are further presented in \cref{tab:HGG_location_characteristics} and \cref{tab:HGG_location_tumor_characteristics}.

\begin{figure}
    \centering
    \resizebox{0.9\textwidth}{!}{\subimport{Figures/}{inclusion_flowchart.pgf}}

    \caption{Flow diagram of the inclusion procedure}\label{fig:HGG_location_flowchart}
\end{figure}

\begin{table}
    \sisetup{
    table-number-alignment = center,
    table-text-alignment = center,
    }
\begin{tabular}{L{5cm} S[table-number-alignment=center-decimal-marker, table-format=3.0] S[table-number-alignment=center-decimal-marker, table-format=3.1]}
    \toprule
    & {\thead{N}} & {\thead{\si{\percent}}}\\
    \midrule

    \textbf{Total} & 436 & 100.0\\
    \\

    \textbf{Sex}\\
    \hspace{1em}Male & 276 & 63.3\\
    \hspace{1em}Female & 160 & 36.7\\

    \textbf{Age (years)}\\
    \hspace{1em}$\leq 65$ & 227 & 52.1\\
    \hspace{1em}$> 65$  & 209 & 47.9\\
    \hspace{1em}Mean (SD) & \multicolumn{2}{c}{\hphantom{0}61.5 (16.2)}\\

    \textbf{\acrshort{KPS}}\\
    \hspace{1em}$\leq 70$ & 142 & 32.6\\
    \hspace{1em}$> 70$ & 294 & 67.4\\
    \hspace{1em}Mean (SD) & \multicolumn{2}{c}{\hphantom{0}80 (12.5)}\\

    \textbf{Preoperative \acrshort{MRI} scans}\\
    \hspace{1em}\acrshort{T1C} & 436 & 100.0\\
    \hspace{1em}\acrshort{T2} & 346 & 79.4\\
    \hspace{1em}\acrshort{FLAIR} & 90 & 20.6\\

    \textbf{Neurosurgical procedure}\\
    \hspace{1em}Resection & 340 & 78.0\\
    \hspace{1em}Biopsy & 96 & 22.0\\

    \textbf{\acrshort{MGMT} status}\\
    \hspace{1em} Methylated & 211 & 48.4\\
    \hspace{1em} Unmethylated & 225 & 51.6\\
    \bottomrule
\end{tabular}
\caption{Patient and \gls{tumor} characteristics. Abbreviations: \acrcaption{KPS}, \acrcaption{SD}, \acrcaption{T1C}, \acrcaption{T2}, \acrcaption{FLAIR}}\label{tab:HGG_location_characteristics}
\end{table}


\begin{table}[htbp]
    \sisetup{
    table-number-alignment = center,
    table-text-alignment = center,
    }
\begin{tabular}{L{5cm} S[table-number-alignment=center-decimal-marker, table-format=2.1] S[table-number-alignment=center-decimal-marker, table-format=3.1]   S[table-number-alignment=center-decimal-marker, table-format=2.1]  S[table-number-alignment=center-decimal-marker, table-format=2.1] S[table-number-alignment=center-decimal-marker, table-format=0.3]}
    \toprule
    & \multicolumn{2}{c}{\thead{\acrshort{MGMT}\\methylated}} & \multicolumn{2}{c}{\thead{\acrshort{MGMT}\\unmethylated}}\\
    \cmidrule(lr){2-3} \cmidrule(lr){4-5}
    & {Median} & {IQR} & {Median} & {IQR} & {p-value}\\
    \midrule
    Contrast-enhancing & 30.1 & 39.5 & \num{35.0} & \num{45.8} & \num{0.130}\\
    Non-enhancing& 75.5 & 105.0 & 65.5 & 84.2 & 0.338\\
    \\
    Ratio non-enhancing vs.\ contrast-enhancing & 2.5 & 3.3 & 2.1 & 2.6 & 0.045\\
    \bottomrule
\end{tabular}
\caption{Preoperative \gls{tumor} volume (in \si{\cubic\centi\metre}) of different subgroups. Abbreviations: \acrcaption{IQR}}\label{tab:HGG_location_tumor_characteristics}
\end{table}


For visual inspection, heatmaps based on \gls{T1C} and \gls{T2}/\gls{FLAIR} segmentations were created for all 436 patients (\cref{fig:HGG_location_heatmap}) and for the \cgls{MGMT} methylated and \gls{MGMT} unmethylated subgroups (\cref{fig:HGG_location_heatmap_genetic}).
We also created frequency difference maps between \gls{MGMT} methylated vs.\ unmethylated \cgls{glioblastoma} (\cref{fig:HGG_location_frequency_map}).
Visual inspection of the maps in \cref{fig:HGG_location_heatmap} suggests that \cgls{glioblastoma} were most frequently located in the right temporal, insular, and parietal area, and near the periventricular area both frontally and occipitally.
Visual inspection of \cref{fig:HGG_location_heatmap_genetic,fig:HGG_location_frequency_map} indicates that when compared with \gls{MGMT} unmethylated \cgls{glioblastoma}, \gls{MGMT} methylated \cgls{glioblastoma} were more frequently located near bifrontal and left occipital periventricular area (up to \per{6.5} frequency difference) and less frequently near the right occipital periventricular area (up to \per{9.1} frequency difference).

\begin{figure}[htbp]
\centering
\resizebox{\textwidth}{!}{\subimport{Figures/}{heatmap_all_T1_and_T2.pgf}}
\caption{Heatmaps of all 436 \acrshort{IDH}-wildtype glioblastoma}\label{fig:HGG_location_heatmap}
\end{figure}

\begin{figure}[htbp]
    \centering
    \begin{subfigure}[b]{\textwidth}
        \centering
        \resizebox{\textwidth}{!}{\subimport{Figures/}{heatmap_methylated_T1_and_T2.pgf}}
        \caption{Distribution of \gls{MGMT} methylated \glspl{tumor}}\label{fig:HGG_location_heatmap_methylated}
    \end{subfigure}
    \begin{subfigure}[b]{\textwidth}
        \centering
        \resizebox{\textwidth}{!}{\subimport{Figures/}{heatmap_unmethylated_T1_and_T2.pgf}}
        \caption{Distribution of \gls{MGMT} unmethylated \glspl{tumor}}\label{fig:HGG_location_heatmap_unmethylated}
    \end{subfigure}
    \caption{Heatmaps of \acrshort{MGMT} methylated (\numbersamples{= 211}) and unmethylated (\numbersamples{= 225}) \cgls{glioblastoma}}\label{fig:HGG_location_heatmap_genetic}
\end{figure}


\begin{figure}[htbp]
    \centering
    \resizebox{\textwidth}{!}{\subimport{Figures/}{frequency_map_T1_and_T2.pgf}}
    \caption{Frequency difference maps between \acrshort{MGMT} methylated (\numbersamples{= 211}) and unmethylated (\numbersamples{= 225}) \cgls{glioblastoma}}\label{fig:HGG_location_frequency_map}
\end{figure}

To test whether this difference was statistically significant, voxel-wise analyses of both the \gls{T1C} and \gls{T2}/\gls{FLAIR} segmentation heatmaps were performed.
Although statistical analysis of the \gls{T1C} scans marked a region near the right occipital periventricular area as a potentially discriminating area between \gls{MGMT} methylated vs.\ unmethylated \cgls{glioblastoma}, this difference was not statistically significant (\cref{fig:HGG_location_pvalue_map}, together with corresponding p-values).
\cref{fig:HGG_location_pvalue_map} in fact shows that not any statistically significantly discriminating brain area between \gls{MGMT} methylated and unmethylated \cgls{glioblastoma} could be found (\pvalue{< 0.05}).
This result did not change after an additional analysis with correction for age as potential confounding factor.

\begin{figure}[htbp]
    \centering
    \begin{subfigure}[b]{\textwidth}
        \centering
        \resizebox{\textwidth}{!}{\subimport{Figures/}{p_value_methylated_T1_and_T2.pgf}}
        \caption{P-values of \gls{MGMT} methylated \glspl{tumor}}\label{fig:HGG_location_pvalue_methylated}
    \end{subfigure}
    \begin{subfigure}[b]{\textwidth}
        \centering
        \resizebox{\textwidth}{!}{\subimport{Figures/}{p_value_unmethylated_T1_and_T2.pgf}}
        \caption{P-values of \gls{MGMT} unmethylated \glspl{tumor}}\label{fig:HGG_location_pvalue_unmethylated}
    \end{subfigure}
    \caption{P-value maps of \acrshort{MGMT} methylated (\numbersamples{= 211}) and unmethylated (\numbersamples{= 225}) glioblastoma}\label{fig:HGG_location_pvalue_map}
\end{figure}

\ifdigital
    We have also created scroll-through animations which show all slices of the location heatmaps (\cref{fig:HGG_local_heatmaps_full_all,fig:HGG_local_heatmaps_subgroups}), frequency difference maps (\cref{fig:HGG_loc_freq_map_full}), and p-value maps (\cref{fig:HGG_loc_p_value_map_full}).
\else
    We have also created scroll-through animations which show all slices of the location heatmaps, frequency difference maps, and p-value maps. These scroll-through animations are only available in the digital version of this thesis.
\fi


\section{Discussion}

This study voxel-wise analyzed \gls{T1C} and \gls{T2}/\gls{FLAIR} heatmaps and showed that there was no statistically significant difference in anatomical localization between \gls{MGMT} methylated vs.\ unmethylated \gls{IDH}-wildtype \cgls{glioblastoma}.

The primary reason to initiate this study was to re-evaluate the anatomic localization of \gls{MGMT} methylated vs.\ unmethylated \cgls{glioblastoma} in light of the updated \gls{WHO} 2016 classification era following conflicting reports on this topic \autocite{smits2017imaging}.
\citeauthorref{ellingson2013probabilistic} reported that \cgls{glioblastoma} with \gls{MGMT} methylation were lateralized to the left hemisphere (temporal lobe) and that those without were lateralized to the right hemisphere, which was in line with their previous article in which they included a substantial portion of their previously studied \cgls{glioblastoma} population \autocite{ellingson2012anatomic}.
However, in contrast to these findings, there are also studies that found the reverse pattern of hemispheric lateralization, in which \cgls{glioblastoma} with \gls{MGMT} methylation were located more frequently in the right hemisphere, while those without \gls{MGMT} methylation lateralized to the left hemisphere \autocite{wang2014anatomical}.
Additionally, there are conflicting reports on lobar distribution, in which \cgls{glioblastoma} with \gls{MGMT} methylation were more frequently located in the parietal and occipital lobes, while those without were located more frequently in the temporal lobes \autocite{eoli2007methylation}.
A recent study suggested after qualitative analyses that subventricular zones were more frequently spared with \gls{MGMT} methylated \cgls{glioblastoma}, but found no difference in hemispheric lateralization between \cgls{glioblastoma} with and without \gls{MGMT} promoter methylation \autocite{han2018structural}.
Finally, there are also studies that report no differences in localization between \cgls{glioblastoma} with and without \gls{MGMT} methylation in concordance with the findings of our study \autocite{carillo2012relationship, drabycz2010analysis}.

These conflicting results in the literature can potentially be ascribed to two methodological issues.
First, inconsistencies may arise from variations in \cgls{glioblastoma} patient populations across studies, many of which were performed in the pre-\gls{WHO} 2016 classification era when the impact of molecular subtyping of \cgls{glioblastoma} according to \gls{IDH} mutation status was less of a consideration \autocite{louis20162016}.
\citeauthorref{ellingson2013probabilistic} included a series of 507 de novo \cgls{glioblastoma} with mixed \gls{IDH} subtypes, including 366 \gls{IDH}-wildtype, 34 \gls{IDH}-mutated \cgls{glioblastoma}, and also 107 \cgls{glioblastoma} without data on \gls{IDH} mutation status.
Moreover, the majority of the studies did not report the \gls{IDH} mutation status of included \cgls{glioblastoma} \autocite{ellingson2012anatomic,wang2014anatomical,eoli2007methylation,drabycz2010analysis}.

Mixing molecular subtypes or not knowing the \gls{IDH} mutation status of \cgls{glioblastoma} is undesirable when assessing topographical distribution of molecular subtypes \autocite{louis20162016}, since it is now known that \gls{IDH}-mutated \cgls{glioblastoma} represent a distinct molecular subtype of \cgls{glioblastoma} from a distinct precursor lesion which have a predominantly frontal lobe involvement when compared with \gls{IDH}-wildtype \cgls{glioblastoma} \autocite{lai2011evidence}.
This topographic link between \gls{IDH} mutation and \gls{MGMT} methylation was also suggested by \citeauthorref{ellingson2013probabilistic} demonstrating that \gls{IDH}-mutated and \gls{MGMT} methylated \cgls{glioblastoma} were indeed more frequently localized in the frontal lobe.
This has not only been demonstrated in \cgls{glioblastoma} but also in non-enhancing low grade glioma in which \gls{IDH}-mutated low grade glioma (both \gls{oli} and \gls{astro}) were more frequently located in the frontal lobes, while non-enhancing \gls{IDH}-wildtype astrocytoma were more frequently located in the basal ganglia of the right hemisphere \autocite{chaichana2011factors}.
This topographical link thus suggests \gls{IDH} mutation status as a (confounding) factor between \gls{MGMT} methylation status and localization.
Therefore, studies must be conducted based on homogeneous \gls{tumor} populations with respect to \gls{IDH} mutational status.
This hypothesis was recently supported by \citeauthorref{roux2019mri}, who assessed a homogenous \gls{IDH}-wildtype \cgls{glioblastoma} population (\numbersamples{= 392}) and found no difference in localization between \cgls{glioblastoma} with and without \gls{MGMT} methylation, in line with our study.

Second, the conflicting results in the literature may arise from different statistical methods that were used across studies.
Studies often investigated the anatomic localization of \cgls{glioblastoma} with and without \gls{MGMT} promoter methylation with visual examination, qualitatively, without a statistical, voxel-wise quantitative analysis \autocite{carillo2012relationship,eoli2007methylation,han2018structural,drabycz2010analysis}.
\citeauthorref{ellingson2013probabilistic} used frequency difference maps to demonstrate that \gls{MGMT} methylated \cgls{glioblastoma} were more frequently localized in the left temporal lobe.
Using similar frequency difference maps, we also found topographical differences, which indicated that when compared with \gls{MGMT} unmethylated \cgls{glioblastoma}, \gls{MGMT} methylated \cgls{glioblastoma} were more frequently localized near bifrontal and right occipital periventricular area and less frequently near the right occipital periventricular area.
However, we showed that these apparent differences did not survive rigorous statistical testing.
\citeauthorref{ellingson2012anatomic} report the use of \say{Analysis of Differential Involvement} for their statistical analysis, which is based on the Fisher exact test.
We used \gls{FSL} randomise, which is different from the Fisher exact test because it does not make any assumptions about the underlying distribution of the variables \autocite{winkler2014permutation}.
Another methodological difference can be found in the correction for multiple comparisons.
\citeauthorref{ellingson2012anatomic} used random permutations based on \citeauthorref{bullmore1999global} instead of the more recently proposed and widely accepted method of doing random permutations employed in \gls{FSL} randomise \autocite{winkler2014permutation}.
Furthermore, the method by \citeauthorref{bullmore1999global} requires a user-defined threshold for clustering, which can impact the results substantially \autocite{bullmore1999global}.
Instead, we used threshold free cluster enhancement, which does not require thresholding to determine the clusters, and which has been shown to have a higher sensitivity compared to other methods \autocite{smith2009threshold}.
Our stringent methodology of rigorous statistical testing and applying new insights in \cgls{glioblastoma} molecular subtyping to a large studied patient population are the strengths of our study.

\section{Limitations}

The main limitation of this study is its retrospective design, which may have introduced selection and confounding biases.
Selection bias may occur when patients who receive diagnostic biopsies are excluded from analysis, since these \glspl{tumor} are often large, multifocal, located deep within the basal ganglia, or crossing midline.
This may skew the results on \gls{tumor} localization of \cgls{glioblastoma}, which is our main outcome.
We have therefore attempted to limit this bias first by consecutive inclusion of all \cgls{glioblastoma} patients operated upon between 2011 and 2018 in our cohort, including diagnostic biopsies.
In addition, it is known that \gls{tumor} localization is associated with \gls{IDH} mutation status, with \gls{IDH}-mutated \glspl{tumor} located more frequently in the frontal lobes, as mentioned earlier \autocite{lai2011evidence}.
Since \gls{IDH} mutation status is both associated with \gls{tumor} localization and \gls{MGMT} methylation status, it may function as a confounding factor.
We therefore have also attempted to limit this potential bias by excluding all \gls{IDH}-mutated \glspl{tumor}.
Another limitation is that we included patients from two medical centers from a period of over seven years.
This introduced variation of \gls{MRI} scan protocols such as magnet strength, voxel size, and slice thickness, which consequently may have negatively influenced registration accuracy and anatomical localization.
Such registration inaccuracies can however be considered minor relative to the size of the \gls{tumor}, and it is therefore unlikely that our results were significantly impacted by scanner variations.
Additionally, \gls{tumor} volume assessment on these \gls{MRI} scans were performed by one observer without confirmation of a second, independent assessor.
This may have introduced some degree of information bias.
We have attempted to limit this bias during volumetric assessment by blinding the assessor for patients' clinical and molecular characteristics.
Also, it is known that both the inter- and intraobserver agreement for preoperative \gls{tumor} volumes in \cgls{glioblastoma} is relatively high, while small variations in segmentation will probably have only a very limited effect on determining gross \gls{tumor} localization \autocite{kubben2010intraobserver}.
Finally, it should be noted that the known intertest variability is a limitation of \gls{MGMT} analyses, as assays used in other studies may produce slightly different \gls{MGMT} methylation results \autocite{wick2014mgmt}.
This may partially explain the variety in the proportion of \gls{MGMT} methylated \glspl{tumor} reported in literature.

To conclude, in the largest homogenous \gls{IDH}-wildtype \cgls{glioblastoma} population to date, we showed that visual appearance of differences could not be confirmed with rigorous voxel-wise statistical testing and thus that there is no statistical difference in anatomical localization between \gls{IDH}-wildtype \cgls{glioblastoma} with vs.\ without \gls{MGMT} promoter methylation.

\section*{Acknowledgements}

The authors would like to thank Claudine Nogarede-Bloemendaal for assistance with data collection at the Haaglanden MC\@.

\ifdigital
    \newpage
    \begin{subappendices}
        \section{Heatmaps all patients}
        \begin{figure}[H]
            \centering
            \begin{subfigure}[t]{0.4\textwidth}
                \centering
                \animategraphics[controls,width=\textwidth,palindrome]{10}{Figures/Animation/heatmaps_all_T1/slice-}{0}{149}
                \caption{\acrshort{T1C} heatmap, all patients}\label{fig:HGG_loc_T1_heatmap_all}
            \end{subfigure}
            \hfill
            \begin{subfigure}[t]{0.4\textwidth}
                \centering
                \animategraphics[controls,width=\textwidth,palindrome]{10}{Figures/Animation/heatmaps_all_T2/slice-}{0}{149}
                \caption{\acrshort{T2}/\acrshort{FLAIR} heatmap, all patients}\label{fig:HGG_loc_T2_heatmap_all}
            \end{subfigure}
            \caption{Animated scroll-through of location heatmaps of the \glspl{tumor} for all patients}\label{fig:HGG_local_heatmaps_full_all}
        \end{figure}

        \newpage

        \section{Heatmaps MGMT methylation subgroups}
        \begin{figure}[H]
            \centering
            \begin{subfigure}[t]{0.4\textwidth}
                \centering
                \animategraphics[controls,width=\textwidth,palindrome]{10}{Figures/Animation/heatmaps_methylated_T1/slice-}{0}{149}
                \caption{\acrshort{T1C} heatmap, \gls{MGMT} methylated patients}\label{fig:HGG_loc_T1_heatmap_methylated}
            \end{subfigure}
            \hfill
            \begin{subfigure}[t]{0.4\textwidth}
                \centering
                \animategraphics[controls,width=\textwidth,palindrome]{10}{Figures/Animation/heatmaps_methylated_T2/slice-}{0}{149}
                \caption{\acrshort{T2}/\acrshort{FLAIR} heatmap, \gls{MGMT} methylated patients}\label{fig:HGG_loc_T2_heatmap_methylated}
            \end{subfigure}

            \begin{subfigure}[t]{0.4\textwidth}
                \centering
                \animategraphics[controls,width=\textwidth,palindrome]{10}{Figures/Animation/heatmaps_unmethylated_T1/slice-}{0}{149}
                \caption{\acrshort{T1C} heatmap, \acrshort{MGMT} unmethylated patients}\label{fig:HGG_loc_T1_heatmap_unmethylated}
            \end{subfigure}
            \hfill
            \begin{subfigure}[t]{0.4\textwidth}
                \centering
                \animategraphics[controls,width=\textwidth,palindrome]{10}{Figures/Animation/heatmaps_unmethylated_T2/slice-}{0}{149}
                \caption{\acrshort{T2}/\acrshort{FLAIR} heatmap, \acrshort{MGMT} unmethylated patients}\label{fig:HGG_loc_T2_heatmap_unmethylated}
            \end{subfigure}
            \caption{Animated scroll-through of location heatmaps of the \glspl{tumor} for the MGMT methylated and MGMT unmethylated subgroups}\label{fig:HGG_local_heatmaps_subgroups}
        \end{figure}


        \newpage
        \section{Frequency difference maps}
        \begin{figure}[H]
        \centering
        \begin{subfigure}[t]{0.4\textwidth}
            \centering
            \animategraphics[controls,width=\textwidth,palindrome]{10}{Figures/Animation/frequency_maps_T1/slice-}{0}{149}
            \caption{\gls{T1C} frequency difference map}\label{fig:HGG_loc_T1_freq_dif_gif}
        \end{subfigure}
        \hfill
        \begin{subfigure}[t]{0.4\textwidth}
            \centering
            \animategraphics[controls,width=\textwidth,palindrome]{10}{Figures/Animation/frequency_maps_T2/slice-}{0}{149}
            \caption{\gls{T2}/\gls{FLAIR} frequency difference map}\label{fig:HGG_loc_T2_freq_dif_gif}
        \end{subfigure}
        \caption{Animated scroll-through of the frequency difference maps between \gls{MGMT} methylated and \gls{MGMT} unmethylated \gls{GBM}}\label{fig:HGG_loc_freq_map_full}
        \end{figure}

        \newpage
        \section{P-value maps}
        \begin{figure}[H]
            \centering
            \begin{subfigure}[t]{0.4\textwidth}
                \centering
                \animategraphics[controls,width=\textwidth,palindrome]{10}{Figures/Animation/p_value_T1_methylated/slice-}{0}{149}
                \caption{\acrshort{T1C} p-value map, \acrshort{MGMT} methylated patients}\label{fig:HGG_loc_T1_p_value_map_methylated}
            \end{subfigure}
            \hfill
            \begin{subfigure}[t]{0.4\textwidth}
                \centering
                \animategraphics[controls,width=\textwidth,palindrome]{10}{Figures/Animation/p_value_T2_methylated/slice-}{0}{149}
                \caption{\acrshort{T2}/\acrshort{FLAIR} p-value map, \acrshort{MGMT} methylated patients}\label{fig:HGG_loc_T2_p_value_map_methylated}
            \end{subfigure}

            \begin{subfigure}[t]{0.4\textwidth}
                \centering
                \animategraphics[controls,width=\textwidth,palindrome]{10}{Figures/Animation/p_value_T1_unmethylated/slice-}{0}{149}
                \caption{\acrshort{T1C} p-value map, \acrshort{MGMT} unmethylated patients}\label{fig:HGG_loc_T1_p_value_map_unmethylated}
            \end{subfigure}
            \hfill
            \begin{subfigure}[t]{0.4\textwidth}
                \centering
                \animategraphics[controls,width=\textwidth,palindrome]{10}{Figures/Animation/p_value_T2_unmethylated/slice-}{0}{149}
                \caption{\acrshort{T2}/\acrshort{FLAIR} p-value map, \acrshort{MGMT} unmethylated patients}\label{fig:HGG_loc_T2_p_value_map_unmethylated}
            \end{subfigure}
            \caption{Animated scroll-through of the p-value maps of the \glspl{tumor}}\label{fig:HGG_loc_p_value_map_full}
        \end{figure}
    \end{subappendices}
\fi
